lib/namedQuery/namedQuery.client.js

```diff
@@ -75,6 +75,14 @@ export default class extends Base {
     }
 
     /**
+     * Gets the count of matching elements in sync.
+     * @returns {any}
+     */
+    async getCountSync() {
+        return await callWithPromise(this.name + '.count', prepareForProcess(this.body, this.params));
```

Comment:
You are right. I overlooked the _async suffix in the fetchSync methods.

On the other hand: It seems to work with the default count method and I don't see why it should be required to differentiate between sync/promise and async calls anyway.

I don't see why the Meteor method must return a Promise too giving the way how you implemented `callWithPromise`